src/main/java/com/google/sps/data/Recommended.java

```diff
@@ -0,0 +1,68 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Recommended {
+
+  public static Drink getDrink(String storeid, String drinkName) {
+    List<Drink> drinklist = DrinkDAO.getDrinksByStore(storeid);
+    for (Drink d : drinklist) {
+      if (d.getName().equals(drinkName)) {
+        return d;
+      }
+    }
+    return null;
+  }
+  // given a coffeeshop, return its tydrecommended score
+  public static double getScore(Drink d) {
+    double comments = d.getNumRatings();
```

Comment:
1) Does "comments" refer to the total number of comments?
2) If yes, should we rename the method from getNumRatings() to getNumComments()?
3) What's the rationale for storing comments in a floating point, instead of an integer or long?

Reply:
the number of comments and the number of ratings will always be the same due to our form process; i have changed comments to numComments for a more appropriate name, but Drink.java belongs to Deron and i don't want to mess up any one else's code. Storing the comments in a floating point was just a personal preference because i would be having to use this type at the end of my calculation regardless, but i have changed it to int because this variable itself will not ever have a decimal.